apps/homework/models.py

```diff
@@ -37,6 +37,11 @@ class Definition(models.Model):
     max_submissions_per_student = models.IntegerField(default=20, null=False, validators=[MaxValueValidator(40), MinValueValidator(0)])
     force_github = models.BooleanField(default=False)
 
+    # Jupyter Notebook Grading Parameters
+    jupyter_notebook_enabled = models.BooleanField(default=False)
+    jupyter_notebook_lowest = models.FloatField(default=0.0, null=True, blank=False)
+    jupyter_notebook_highest = models.FloatField(default=1.0, null=True, blank=False)
```

Comment:
I thought homework's already had a high/low score? I checked the `Definition` model and it looks like we don't, though?

Reply:
Can add criteria with high and low scores, but they don't inherently exist on homework definitions. I added this rather than automatically creating a criterion, so it is uniquely identifiable and easy to access.